apps/demo-react-cra/package.json

```diff
@@ -43,5 +40,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@types/jest": "^29.5.12"
```

Comment:
We moved from `jest` to `vitest` — do we still need `jest` types?

Reply:
Actually @arboleya it seems we are [still using jest in [demo-react-cra](https://github.com/FuelLabs/fuels-ts/actions/runs/8695553192/job/23846846384?pr=2026#step:6:36)

Perhaps we should address this in a follow up PR?

Reply:
My bad - I was assuming it was the root `package.json`.

Nothing to do here.